packages/backend/src/managers/playgroundV2Manager.ts

```diff
@@ -23,21 +23,62 @@ import type { ModelOptions } from '@shared/src/models/IModelOptions';
 import type { Stream } from 'openai/streaming';
 import { ConversationRegistry } from '../registries/conversationRegistry';
 import type { Conversation, PendingChat, UserChat } from '@shared/src/models/IPlaygroundMessage';
-
-export class PlaygroundV2Manager implements Disposable {
+import type { PlaygroundV2 } from '@shared/src/models/IPlaygroundV2';
+import { Publisher } from '../utils/Publisher';
+import { Messages } from '@shared/Messages';
+import type { ModelInfo } from '@shared/src/models/IModelInfo';
+import { withDefaultConfiguration } from '../utils/inferenceUtils';
+
+export class PlaygroundV2Manager extends Publisher<PlaygroundV2[]> implements Disposable {
+  #playgrounds: Map<string, PlaygroundV2>;
```

Comment:
We might prefer to add a `modelId` to the Conversation object rather than creating a new store ?

Reply:
I have superposed our 2 logics without merging them for now, it will be part of the next PR